### DISCUSSION TITLE
This domain is new or has not been reviewed yet.

### DISCUSSION THREAD

**Discussion by Amnamot** [phantom/docs#80] [General]
Hello!

When testing payment, Phantom issued the following warning: 
This domain is new or has not been reviewed yet. Proceed with caution.

Please tell me what to do to get rid of this message?

**Comment by adamdelphantom** [comment]
Would you mind providing your dApp information like the URL, any relevant Program IDs and other relevant links like social media, marketing site etc, please?

**Reply by bfriel** [reply]
In general, we will display this message for all domains that are <24 hours old. This can be overridden if the above information is shared.

**Reply by Amnamot** [reply]
Hello!
Due to the fact that the site is still in test mode, albeit in the mainnet, I cannot publicly distribute such links. I will send you a private message.

**Reply by Amnamot** [reply]
I can't send a private message? OK!
Here are the links to the website and Twitter:
https://azagnat.top/
https://twitter.com/JeromLeprechaun
MINTING DOES NOT WORK NOW! ALL COINS SPENT BEFORE THE OFFICIAL RELEASE WILL BE REFUNDED MINUS THE COMMISSION.

**Reply by adamdelphantom** [reply]
@Amnamot I believe the warnings have been removed. Please let us know if issues persist.

**Reply by Amnamot** [reply]
Yes, I didn't notice them today. Thank you!
